1.1/funcsrules.ttl

```diff
@@ -1,3 +1,5 @@
+BASE <http://www.opengis.net/def/geosparql/funcsrules> # baseuri added as per issue 147 
```

Comment:
This is SPARQL syntax. Is this intentional?

Reply:
SPARQL syntax was used for the baseuri as SPARQL syntax had been used for the existing prefix statements. Agreed this is different from all the other ttl files, but either syntax (`PREFIX` and `BASE` vs `@prefix` and `@base`) are valid. Ought we adopt a consistent form across all the ttl files?

Reply:
Well it took me a couple of minutes to figure out why it was different, and I'm an RDF regular! 
I think it would be better if we standardized to the Turtle syntax in Turtle files (even if SPARQL works as well)

Reply:
Agreed. I've amended all the prefix statements and the base statement in funcsrules.ttl to use Turtle syntax. So now all Turtle files in the repo are consistent on this. 
`@prefix prefixName <uri> .` rather than `PREFIX prefixName <uri>` 
and 
`@base <uri> .` rather than `BASE <uri>`

Reply:
> better if we standardized to the Turtle syntax in Turtle files 

From [the Turtle 1.1 spec](https://www.w3.org/TR/turtle/#h_note_1): "The case-insensitive 'PREFIX' and 'BASE' forms were added to align Turtle's syntax with that of SPARQL.".

I prefer this syntax as it's far easier to copy and paste Turtle to SPARQL and back. Particularly apt in GeoSPARQL given the query + RDF modelling nature of the standard.

RDF tools out there seen to handle this syntax except for... TopBraid composer! So I undertand why @dr-shorthair doesn't use it...

Fair enough to change to the (horrible, old, nasty) `@prefx` style for wider use.